### PR TITLE
Improve the documentation for try/1

### PR DESCRIPTION
**Pull request by whatyouhide**

### CODE REVIEW

File: lib/elixir/lib/kernel/special_forms.ex [hunk 1]
```diff
@@ -1707,55 +1713,63 @@ defmodule Kernel.SpecialForms do
...
+    catch
+      :exit, value
+        IO.puts "Exited with value #{inspect(value)}"
+    end
+
+    try do
+      exit(:shutdown)
+    catch
+      kind, value when kind in [:exit, :throw] ->
+        IO.puts "Exited with or thrown value #{inspect(value)}"
```
Comment (ericmj) [inline lib/elixir/lib/kernel/special_forms.ex]: Missing word?

Comment (whatyouhide) [inline lib/elixir/lib/kernel/special_forms.ex]: I think it means `exited with #{inspect(value)} or thrown value #{inspect(value)}`, without repeating the inspected value 😄

Comment (ericmj) [inline lib/elixir/lib/kernel/special_forms.ex]: It reads weirdly, can you fix?

Comment (whatyouhide) [inline lib/elixir/lib/kernel/special_forms.ex]: Yep :) Updated.

**Pull request merged**